release/index.d.ts

```diff
@@ -0,0 +1,9 @@
+import { DataType } from "./src/datatype";
```

Comment:
Cause now it's point back at the original type script sources rather than definition files in `release/datatype`